src/App.js

```diff
@@ -11,13 +11,20 @@ import Layout from './Layout';
 import HomePage from './pages/HomePage';
 import MyReservations from './pages/MyReservations';
 import AddGame from './pages/AddGame';
+import GamesList from './pages/GamesList';
+import dispatchGetUser from './store/slices/get_user_slice';
 
 function App() {
   const dispatch = useDispatch();
+  const user = localStorage.getItem('userInfo');
   useEffect(() => {
     dispatchGetGames(dispatch);
   }, []);
-  const user = localStorage.getItem('userInfo');
+  useEffect(() => {
+    if (user) {
+      dispatchGetUser(dispatch, user.username);
+    }
+  }, [user]);
   return (
```

Comment:
- [OPTIONAL] It would be better to move this functionality to a component instead of the App itself because App usually stored the routes structure and global state.

<img src="https://media3.giphy.com/media/eSwGh3YK54JKU/giphy.gif"/>